src/config.rs

```diff
@@ -35,6 +42,15 @@ pub struct Config {
     pub token: Option<String>,
 }
 
+pub fn merge_configs(first: PdbConfigSection, second: PdbConfigSection) -> PdbConfigSection {
```

Comment:
I'll make a ticket for doing this in acceptance tests, but I want to wait on getting the FOSS and PE module PRs for configuring that stuff out of the box by default before adding those tests.